mods/cnc/maps/gdi05a/rules.yaml

```diff
@@ -4,7 +4,7 @@ World:
 	MusicPlaylist:
 		StartingMusic: rain
 	MissionData:
-		Briefing: A GDI field base is under attack. They have fended off one attack but will not survive another.\n\nMove to the base, repair the structures and then launch a strike force to destroy the Nod base in the area.\n\nDestroy all Nod units and structures.
+		Briefing: A GDI field base is under attack. They have fended off one attack but will not survive another.\n\nMove to the base, repair the structures, and launch a strike force to destroy the Nod base in the area.\n\nDestroy all Nod units and structures.
```

Comment:
", and launch" -> ", then launch" to avoid changing meaning. Likewise for the other two GDI05 variants.